hw4.js

```diff
@@ -0,0 +1,475 @@
+// ----==== JS School - Lecture 4 HW ====----
+
+/**
+ * Ð¡riteria for assessment
+ *
+ * 5 - All tasks are correctly solved (23 items), the code is clean, the solutions are optimal;
+ * 4 - Correctly solved all the tasks of the base block (15 tasks), the code is clean;
+ * 3 - At least 10 problems are correctly solved, the code is clean;
+ * 2 - Correctly solved at least 10 problems;
+ * 1 - At least 5 problems solved correctly.
+ */
+
+/**
+ * Warning
+ *
+ * Do not rename function names or change arguments.
+ */
+
+// ----==== Basic exercises (15 items) ====----
+/**
+ * Exercise 1
+ *
+ * Write a function that returns odd array values.
+ * [1,2,3,4] => [1,3]
+ */
+let arr1 = [1, 2, 3, 4];
+const getOddValues = (numbers) => {
+    let result = [];
+    for (let i = 0; i < arr1.length; i++) {
+        if (i % 2 === 0) {
+            //continue;
+            result.push(arr1[i]);
+        }
+    }
+    return result;
+};
+
+/**
+ * Exercise 2
+ *
+ * Write a function that returns the smallest value of an array
+ * [4,2,10,27] => 2
+ */
+let arr2 = [4, 2, 10, 27];
+const getSmallestValue = (numbers) => {
+    let result = 0;
```

Comment:
let minValue = numbers[0]

let's set the first item of array like an initial value